docs/reference/cluster/tasks.asciidoc

```diff
@@ -184,7 +184,7 @@ The new `description` field contains human readable text that identifies the
 particular request that the task is performing such as identifying the search
 request being performed by a search task like the example above. Other kinds of
 task have different descriptions, like <<docs-reindex,`_reindex`>> which
-has the search and the destination, or <<docs-bulk,`_bulk`>> which just has the
+has the source and the destination index information, or <<docs-bulk,`_bulk`>> which just has the
```

Comment:
I am not sure about the "index information". The documentation about reindexing says the following:

> The source can be any existing index, alias, or data stream. The destination must differ from the source. [...]
The destination should be configured as wanted before calling _reindex.

Based on this I would omit the index information and I would leave it more open. What do you think?